minikerberos/communication.py

```diff
@@ -248,38 +243,41 @@ def get_TGT(self, override_etype = None, decrypt_tgt = True):
 		kdc_req['msg-type'] = MESSAGE_TYPE.KRB_AS_REQ.value
 		kdc_req['padata'] = [pa_data_1]
 		kdc_req['req-body'] = KDC_REQ_BODY(kdc_req_body)
-		
+
 		req = AS_REQ(kdc_req)	
 		
 		logger.debug('Sending initial TGT to %s' % self.ksoc.get_addr_str())
-		rep = self.ksoc.sendrecv(req.dump(), throw = False)
```

Comment:
Why did the throw variable get removed?

Reply:
Don't we want to throw an exception when the TGT communication fails?

Reply:
Initially not. At the very first occasion a kerberos error could happen because it might ask for pre-auth. Currently the lib doesn't have an option to "force" pre-auth as some users might don't even need preauth (yes, I know it's unwise to be turned off but the standard allows it)